pkg/api/install/install_test.go

```diff
@@ -57,7 +57,7 @@ func TestCodec(t *testing.T) {
 	if err := json.Unmarshal(data, &other); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	if other.APIVersion != latest.GroupOrDie("").Version || other.Kind != "Pod" {
+	if other.APIVersion != latest.GroupOrDie("").GroupVersion.Version || other.Kind != "Pod" {
```

Comment:
Sure, I agree hardcoding package dependency makes sense if a package depends on a specific version, but most times this is not the case. Let's deal with it when we hit such a case.